editor/src/com/talosvfx/talos/editor/addons/scene/widgets/gizmos/MapGizmo.java

```diff
@@ -28,22 +28,6 @@ public void setSelected (boolean selected) {
 
     private void setupForCustomGrid (GameObject gameObject) {
 
-        SceneEditorWorkspace.GridProperties gridProperties = SceneEditorWorkspace.getInstance().gridProperties;
```

Comment:
what was this for? :D

Reply:
It is a hook to let us know when the map is selected. So that we can change modes and configure the grid properties based on the map we have selected